### DISCUSSION TITLE
Provide an `.RPM` installer for Fedora.

### DISCUSSION THREAD

**Discussion by beedell-roke** [kraxarn/spotify-qt#275] [Ideas]
It isn't mentioned in [`blob/91da24f71aa98c9e1761f40f665ee86361a505dd/readme.md#installing`](https://github.com/kraxarn/spotify-qt/blob/91da24f71aa98c9e1761f40f665ee86361a505dd/readme.md#installing). Considering one exists for openSUSE, it shouldn't be as difficult as it usually is.

**Comment by kraxarn** [comment]
I personally don't maintain the OpenSUSE builds, and have very little experience with packaging for Fedora myself.
Maybe Flatpak builds makes more sense for now, which is planned to be added soon.

**Reply by beedell-roke** [reply]
> Maybe Flatpak builds makes more sense for now, which is planned to be added soon.

@kraxarn, one significant problem with them is generally theming, but since you use Qt6, it'll at least be a *better* experience than GTK4. Thanks.

**Comment by kraxarn** [comment]
I changed my mind, it's on [Copr](https://copr.fedorainfracloud.org/coprs/kraxarn/spotify-qt) now! :tada:

**Reply by beedell-roke** [reply]
> https://github.com/kraxarn/spotify-qt/discussions/275#discussioncomment-12217092

@kraxarn, thank you, although I immediately encountered [`issues/55#issuecomment-2661590856`](https://github.com/kraxarn/spotify-qt/issues/55#issuecomment-2661590856).
